### PR TITLE
[JBTM-3335] updating jca tomcat dependencies

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.redhat.com/browse/JBTM-3335

With cleaning and updating the dependencies having test working with more recent IronJacamar and Hibernate Validator.

### CODE REVIEW

File: jca-and-tomcat/repo/com/github/fungal/fungal/0.11.0.Final/fungal-0.11.0.Final.pom.sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+e91f98a175adf47dcc1baf86d8134888db61ee75
```
Comment (mayankkunwar) [inline jca-and-tomcat/repo/com/github/fungal/fungal/0.11.0.Final/fungal-0.11.0.Final.pom.sha1:1]: How do you calculate this ID

Comment (ochaloup) [reply jca-and-tomcat/repo/com/github/fungal/fungal/0.11.0.Final/fungal-0.11.0.Final.pom.sha1:1]: as I've already mentioned on in chat the id is `sha1` hash of the file. You can verify if it's placed correctly by running
`sha1sum jca-and-tomcat/repo/com/github/fungal/fungal/0.11.0.Final/fungal-0.11.0.Final.pom`

**Comment by ochaloup** [comment]
@mayankkunwar how does it go. Should I close this PR as you will adapt the changes to your new one? Or will we merge this one and then you will build upon these changes some further fixes?

**Comment by mayankkunwar** [comment]
@ochaloup changes looks good, you may close this PR and merge.

**Comment by ochaloup** [comment]
Ok, I'm merging this. Thanks.

**Pull request merged**